pkg/api/types.go

```diff
@@ -1920,6 +1920,9 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// If specified, the pod's tolerations
+	// +optional
+	Tolerations []Toleration
```

Comment:
My concern is that we need to use `Tolerations ` and `Taint` as pair, but seems there is no comment clarifying this, what do you say @davidopp , thanks.